src/types.ts

```diff
@@ -0,0 +1,44 @@
+import { Domain } from "@zero-tech/data-store-core";
+
+export interface DomainsResponse {
+  numResults?: number;
+  results: Domain[];
+}
+
+export interface DomainResponse {
+  result: Domain | undefined;
+}
+
+export enum DomainSortBy {
+  label = 0,
+  owner = 1,
+}
+export enum DomainSortDirection {
+  desc = 0,
```

Comment:
Why is this `desc = 0` but is translated to `-1` for descending when it is used in the `domainFindOptionsHelper.ts` file?

Reply:
I prefer to have enums start at 0 to give them a default value

This enum is actually unused, so Ill delete it